packages/address/src/graphqlQuery.ts

```diff
@@ -0,0 +1,41 @@
+const query = `
+query countries($locale: SupportedLocale!) {
+  countries(locale: $locale) {
+    name
+    code
+    phoneNumberPrefix
+    address2Key
+    provinceKey
```

Comment:
Add `zipExample` here :)

Reply:
We don't need it so no need to query. :)